server/src/main.ts

```diff
@@ -11,18 +10,14 @@ dotenv.config({ path: __dirname + '/.env' });
 
 async function bootstrap() {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
-  app.setGlobalPrefix('api');
+  app.setGlobalPrefix('/api');
   app.enableCors({
     credentials: true,
     // TODO get env related base url
     origin: 'http://localhost:3000',
   });
   app.useGlobalPipes(new ValidationPipe({ transform: true }));
-  app.useStaticAssets(resolve('./src/public'));
-  app.setBaseViewsDir(resolve('./src/views'));
-  app.setViewEngine('hbs');
   app.use(cookieParser('secret_placeholder'));
```

Comment:
Removing these because we don't use them. These are only needed if we were serving MVC views